lib/transaction/transaction_test.go

```diff
@@ -87,3 +88,12 @@ func TestIsWellFormedTransactionWithInvalidSignature(t *testing.T) {
 	err = tx.IsWellFormed(networkID)
 	require.NotNil(t, err)
 }
+
+func TestIsWellFormedTransactionMaxOperationsInTransaction(t *testing.T) {
+	var err error
+
+	_, tx := TestMakeTransaction(networkID, common.MaxOperationsInTransaction+1)
+	err = tx.IsWellFormed(networkID)
+	require.NotNil(t, err)
+	require.Equal(t, errors.ErrorTransactionHasOverMaxOperations.Code, err.(*errors.Error).Code)
```

Comment:
@Geod24 You mean, the `tx` really has the operations, over limit?

Reply:
I mean that a test which tests a failure should also test a success.
Because if you test that something fails, you cannot be sure why it fails unless you make it pass afterwards.

Reply:
@Geod24 There are already tests for valid transaction in `transaction_test.go`.

Reply:
The point is not to test the valid transaction, it's to make sure that this test fails for the *right* reason.